src/collar/oc_api.lsl

```diff
@@ -333,7 +330,13 @@ UserCommand(integer iAuth, string sCmd, key kID){
         string sCmdx = llToLower(llList2String(lCmd,0));
                 
         if(sCmdx == "channel"){
-            g_iChannel = (integer)llList2String(lCmd,1);
+            integer chan = (integer)llList2String(lCmd,1);
+            if (chan == -1) g_iChannel = -llAbs(g_iChannel);
+            else if (chan == 0) g_iChannel = llAbs(g_iChannel);
+            else if (chan > 0) {
+                if (g_iChannel < 0) g_iChannel = -chan;
+                else g_iChannel = chan;
+            }
```

Comment:
[prefix] channel -1 : disables the collar from listening to commands in local chat,
[prefix] channel 0 : enables the collar to listen to commands in local chat,
[prefix] channel [new number] : change the channel number.

Reply:
Not sure the logic here is correct.  Why inverting the sign with `-chan`?

Reply:
ok, let's leave the negative channel unchanged